LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -191,7 +194,45 @@ public List<Module> GenerateModuleList()
 
         public void CreateClassrooms(int classroomCount)
         {
-            throw new NotImplementedException();
+            foreach(Module module in moduleDictionary.Values)
+            {
+                temporaryModuleHashSet.Add(module);
+
+                Classroom classroom = ObjectGenerator.GenerateClassroom(module);
+
+                classRoomDictionary.Add(Guid.NewGuid(), classroom);
+
+                temporaryModuleHashSet.Remove(module);
+            }
+        }
+
+        public void AssignInstructorsToClassrooms()
+        {
+            foreach(Classroom classroom in classRoomDictionary.Values)
+            {
+                foreach(Instructor instructor in instructorList)
+                {
+                    bool isCandidate = false;
+
+                    foreach (Module module in instructor.ModuleList)
+                    {
+                        if (classroom.Module == module)
+                        {
+                            classroom.Head = instructor;
+
+                            isCandidate = !isCandidate;
```

Comment:
It is unnecessary , isCandidate = true will be sufficient. I would rename in to isCandidateFound.